Content.Shared/GameObjects/Components/Chemistry/SolutionComponent.cs

```diff
@@ -30,6 +30,8 @@ public int MaxVolume
             set => _maxVolume = value; // Note that the contents won't spill out if the capacity is reduced.
         }
 
+        public Solution ContainedSolution => _containedSolution;
```

Comment:
Making the solution object public means that any code can completely bypass the the SolutionComponent functions when adding or removing a reagent. This is a really bad idea, and breaks general OOP encapsulation rules. You need to design it so that outside code has to ask the SolutionComponent for info about the solution, instead of exposing the internal workings of SolutionComponent for outside code to access.